buildenvs/github-action.Dockerfile

```diff
@@ -25,75 +22,14 @@ RUN set -xe; \
         --add safe.directory /go/src/kraftkit.sh/tools/github-action; \
     go build \
         -o /github-action \
-        -tags static \
-        -a \
-        -ldflags='-s -w' \
-        -ldflags '-extldflags "-static"' .;
+        -a;
```

Comment:
Why not static :(

Reply:
![image](https://github.com/unikraft/kraftkit/assets/46280801/5cf64662-de18-4823-9620-f9674d87e2e7)

Lies. But yes, it's not fully static. For the action I suppose it's fine as we provide the whole dockerfile anyways.